composer.json

```diff
@@ -11,15 +11,19 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.0",
+		"php": ">=7.3.0",
```

Comment:
Please hint `^7.4 | ^8.0` since your static analysis passes for `8.0`

Reply:
Good point. I also hinted ^7.3 for temporary backwards compatibility